packages/office-ui-fabric-react/src/components/Layer/Layer.base.tsx

```diff
@@ -52,39 +35,133 @@ export class LayerBase extends BaseComponent<ILayerProps, {}> {
     });
 
     if (this.props.hostId) {
-      if (!_layersByHostId[this.props.hostId]) {
-        _layersByHostId[this.props.hostId] = [];
-      }
+      registerLayer(this.props.hostId, this);
+    }
+  }
+
+  public componentWillMount(): void {
```

Comment:
not sure if we can do without these - but the will* stuff will go away soon.

Reply:
We'll have to adjust these as needed. These changes are part of Chris' original PR where he made fixes to regressions from the original merge.